explore/src/main/scala/explore/targets/TargetSelectionPopup.scala

```diff
@@ -53,6 +54,18 @@ case class TargetSelectionPopup(
 )(using val ctx: AppContextIO)
     extends ReactFnProps(TargetSelectionPopup.component)
 
+object SearchingState extends NewType[Boolean]:
+  val Searching: SearchingState = SearchingState(true)
+  val Idle: SearchingState      = SearchingState(false)
```

Comment:
I have a bit of a conflict with defining `val` constants for `NewType`s since they grow the compiled code. I know it's not 100% the same thing, but maybe we can replace with `inline def`s ?

Reply:
Do you think they will be smaller?